### PR TITLE
Add remove feature

### PR DESCRIPTION
**Pull request by MohamedHNoor**
### In this project, I implement the CRUD (create, read, update, delete) methods. All the elements of the user interface are functional.

- Used webpack to bundle JavaScript
- use proper ES6 syntax
- all changes are saved in the local Storage

### CODE REVIEW

File: src/function.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+const addToDo = (item) => {
+  if (item !== '') {
+    const todo = {
+      description: String,
+      completed: false,
+      index: Number
+    };
+    todo.description = item;
+    todo.index = data.length;
```
Comment (ShahierNashaat) [inline src/function.js]: - [ ] Kindly notice that the property `index` is set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal 5) so the index should be assigned like this.

```javascript
todo.index = data.length + 1;
```

File: src/function.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+};
+
+const deletTask = (task) => {
+  if (task.parentElement !== '') {
+    task.parentElement.remove();
+    data.splice(task.parentElement.index, 1);
+  }
+  data.forEach((task, index) => {
+    task.index = index;
+  });
```
Comment (ShahierNashaat) [inline src/function.js]: - [ ] Also here the tasks' indexes should be assigned like this.

```javascript
data.forEach((task, index) => {
    task.index = index + 1;
});
```

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @MohamedHNoor, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Good overall design ✔️
- Your code is clean and well documented ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @MohamedHNoor , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All linters passed

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**